src/components/dictionaryConcepts/components/CreateMapping.jsx

```diff
@@ -47,7 +56,9 @@ class CreateMapping extends Component {
     }
   }
 
+
   handleKeyPress = (event, inputValue, url) => {
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
Hey! I think it would be greatt to remove the eslint comments from the file

Reply:
It has been fixed.